docs/maridb-backuprestore-from-tempauth.md

```diff
@@ -0,0 +1,161 @@
+  
+  **Note:** This procedure some reference present in <https://github.com/rackerlabs/genestack/blob/55dedc9b4217967e1efc2ec3182e67758cc056d4/docs/infrastructure-mariadb-ops.md?plain=1>
+
+## 4. Manual Restore Using AWS S3 Commands
+  This method retrieves the backup from the overseer and restores it manually, applicable to all production regions as a fallback.
+  
+  ### Steps:
+  #### 1. Access the Region-Specific Overseer:
+          Log in to the overseer node (e.g., ssh user@dfw-prod-overseer-ip or any available method which is allowed to login into DFW Prod in a secured manner).
+  #### 2.Verify AWS CLI Configuration:
+          Ensure the region-specific profile is set up (e.g., dfw_admin for DFW):
+          ```yaml
+          [profile dfw_admin]
+          region = dfw
+          s3 =
+          endpoint_url = https://swift.api.dfw.rackspacecloud.com
+          signature_version = s3v4
+         ```
+         ```yaml
+         [dfw_admin]
+         aws_access_key_id = YOUR_ACCESS_KEY
+         aws_secret_access_key = YOUR_SECRET_KEY
+        ```
+        Adjust for SJC (sjc_admin), IAD (iad_admin) with their endpoints (see table above).
+        Test with below command to list backups:
+        ```shell
+        aws --profile <region>_admin s3 ls s3://mariadb-backups/ .
+        ```
+   #### 3. Retrieve the Backup:
+        List available backups: aws --profile <region>_admin s3 ls s3://mariadb-backups/cron/.
+        Download a specific backup:
+        As a example given here:
+        ```shell
+         aws --profile dfw_admin s3 cp s3://mariadb-backups/cron/backup.2025-02-04T19:05:57Z.gzip.sql /tmp/backup.2025-02-04T19:05:57Z.gzip.sql
+        ```
+        Note: Need to replace the particular file to download as per the requirement. 
+   #### 4. Restore the Backup:
+        Access a test MariaDB instance (e.g., via kubectl exec or a local DB): mysql -u user -p < backup.2025-02-04T19:05:57Z.gzip.sql.
+  #### 5. Single Database Restore:
+        If the backup contains multiple databases, extract the desired database (e.g., nova) using a tool like sed or mysql filters, then restore: mysql -u user -p nova < nova_backup.sql.
+  #### 6. Verify:
+        Check the return code: echo $? (0 indicates success).
+        Query the database: mysql -u user -p -e "SELECT COUNT(*) FROM <table_name>;".
+    
+  **Notes:**
+     Ensure the overseer has network access to the region-specific Swift endpoint.
+
+ **Before applying/executing into Production, First need to apply these commands on DEV or staging environment.**
+
+## 5. References
+  ### https://github.com/mariadb-operator/mariadb-operator/blob/main/docs/BACKUP.md
+  ### https://docs.rackspacecloud.com/storage-object-store-s3-cli/
+  ### https://mariadb.com/docs/server/server-usage/backup-and-restore/backup-and-restore-overview
+ 
+## 6. Escalation:
+  If validation fails, coordinate with James Denton or Jorge Perez to resolve network or any configuration issues.
```

Comment:
We probably shouldn't be calling out rackers in the public docs here. Is this supposed to be internal runbook?